components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/SynapseAttributesApi.java

```diff
@@ -0,0 +1,50 @@
+package org.wso2.carbon.apimgt.internal.service;
+
+import org.wso2.carbon.apimgt.internal.service.dto.ErrorDTO;
+import org.wso2.carbon.apimgt.internal.service.dto.SynapseAttributesDTO;
+import org.wso2.carbon.apimgt.internal.service.SynapseAttributesApiService;
+import org.wso2.carbon.apimgt.internal.service.impl.SynapseAttributesApiServiceImpl;
+import org.wso2.carbon.apimgt.api.APIManagementException;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import javax.inject.Inject;
+
+import io.swagger.annotations.*;
+import java.io.InputStream;
+
+import org.apache.cxf.jaxrs.ext.MessageContext;
+import org.apache.cxf.jaxrs.ext.multipart.Attachment;
+import org.apache.cxf.jaxrs.ext.multipart.Multipart;
+
+import java.util.Map;
+import java.util.List;
+import javax.validation.constraints.*;
+@Path("/synapse-attributes")
+
+@Api(description = "the synapse-attributes API")
+
+@Produces({ "application/json" })
+
+
+public class SynapseAttributesApi  {
+
+  @Context MessageContext securityContext;
+
+SynapseAttributesApiService delegate = new SynapseAttributesApiServiceImpl();
+
+
+    @GET
+    
+    
```

Comment:
extra line spaces

Reply:
That is an Auto-generated File. We don't need to modify anything there.